src/main/java/graphql/normalized/ExecutableNormalizedOperation.java

```diff
@@ -14,22 +14,30 @@
 
 @Internal
 public class ExecutableNormalizedOperation {
-
+    private final String operationName;
```

Comment:
this is to inconsistent: an operation name without type is not enough. Lets add the operation type also here.

Reply:
Do you mean operation kind as in Query / Mutation or Subscription ?

Reply:
yes, exactly